Userland/Libraries/LibVideo/VP9/ContextStorage.h

```diff
@@ -241,7 +241,7 @@ struct SegmentFeature {
 struct ColorConfig {
     u8 bit_depth { 8 };
     ColorSpace color_space { ColorSpace::Bt601 };
-    ColorRange color_range { ColorRange::Studio };
+    VideoFullRangeFlag video_full_range_flag { VideoFullRangeFlag::Studio };
```

Comment:
Not sure if it's worth changing the name here - in the VP9 spec it's referred to as `color_range` (0="Studio swing", 1="Full swing").

Reply:
Undid this one, thanks!